playbook-website/package.json

```diff
@@ -9,8 +9,8 @@
   "dependencies": {
     "@fortawesome/fontawesome-pro": "6.2.1",
     "@mapbox/mapbox-gl-draw": "^1.4.1",
-    "@powerhome/playbook-icons": "0.0.1-alpha.32",
-    "@powerhome/playbook-icons-react": "0.0.1-alpha.32",
+    "@powerhome/playbook-icons": "0.0.1-alpha.36",
+    "@powerhome/playbook-icons-react": "0.0.1-alpha.36",
```

Comment:
i did some branches of branches a while back which is why i think we're going from 32 to 36 :/